src/main/external-resources/renderers/Mirascreen.conf

```diff
@@ -0,0 +1,29 @@
+#----------------------------------------------------------------------------
+# Auto-generated profile for Mirascreen BFCFA04F
+# Based on DefaultRenderer.conf
+# See DefaultRenderer.conf for a description of all possible configuration options.
+#
+
+RendererName = Mirascreen
+RendererIcon = Mirascreen.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# {friendlyName=Mirascreen BFCFA04F, manufacturer= , modelName=Mirascreen BFCFA04F, modelNumber= , modelDescription=Actions-Micro DLNA DMR Using Auto-Generated UPnP Stack, manufacturerURL=http://actions-micro.com, modelURL=http://actions-micro.com}
+# ============================================================================
+
+UserAgentSearch = 
+UpnpDetailsSearch = Mirascreen BFCFA04F
+
+LoadingPriority = 3
+MediaInfo = true
```

Comment:
If you set MediaInfo to true without specifying any `Supported` lines, it mean everything will be transcoded. Does the renderer not support anything at all?

Reply:
it just works